common/templates/xblock_v2/xblock_iframe.html

```diff
@@ -81,8 +81,11 @@
         href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
   <!-- Capa Problem Editing requires CodeMirror -->
   <link rel="stylesheet" href="{{ lms_root_url }}/static/js/vendor/CodeMirror/codemirror.css">
-  <!-- Built-in XBlocks (and some plugins) depends on LMS CSS -->
-  <link rel="stylesheet" href="{{ lms_root_url }}/static/css/lms-course.css">
```

Comment:
For the editor iframe, the `lms-course.css` caused some issues.

I didn't find where this was used to check the results, but the Unit Outline and the Course View seem okay.

We can't add this conditionally if needed.